CHANGELOG.mkd

```diff
@@ -3,10 +3,23 @@ CHANGELOG
 
 Unreleased
 ----
-- Add exec environment source type. The exec source type allows for the implementation of external environment sources [#1042](https://github.com/puppetlabs/r10k/pull/1042)
+
+
+3.5.0
+-----
+
+- Add exec environment source type. The exec source type allows for the
```

Comment:
oh, are we wrapping now? personally, I don't love wrapping text/markdown, but maybe that's just me.

Reply:
I couldn't tell if there was a strong preference in the doc already and it's a pet peeve of mine.

Reply:
what do you have against wrapping text?

Reply:
personally, I find it harder to read. But I don't care too strongly, since we seem to do both in this file.